configs/xlml/tensorflow/solutionsteam_tf_nightly_supported_config.py

```diff
@@ -57,7 +59,10 @@ def get_tf_resnet_config(
   }
 
   test_name = "tf_resnet_imagenet"
-  tpu_name = create_tpu_name(test_name, tpu_version, tpu_cores)
+  benchmark_id = f"{test_name}-v{tpu_version}-{tpu_cores}"
+  # Add default_var to pass DAG check
+  tpu_name = Variable.get(benchmark_id, default_var=None) if is_pod else "local"
```

Comment:
Are variables scoped to a given dag run?

From the [docs](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/variables.html) : `Variables are global, and should only be used for overall configuration that covers the entire installation; to pass data from one Task/Operator to another, you should use XComs instead.`

Reply:
Yes, it's global variables, so I used benchmark_id. If I remember correctly, the problem for XCOMs is that it needs to be used in a task or an operator. So I haven't found a way to apply in `configs/xlml/tensorflow/solutionsteam_tf_nightly_supported_config.py` yet. Also mentioned in this comment: https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/15#discussion_r1433430158.

Let me know your thoughts!

Reply:
I'm not sure this runs in the right order. The body of `generate_tpu_name` will run at the beginning of a DAG run, and this part will run during DAG construction.

If this does work, we can take it as a temporary hack to get the tests passing. But, please leave a TODO to replace this with an XCOM.

Reply:
Sure, I will leave a TODO and think about how to apply with XCOM in this helper config file. Thank you!